pkgs/top-level/python-packages.nix

```diff
@@ -2772,12 +2772,12 @@ in {
   };
 
   click-log = buildPythonPackage rec {
```

Comment:
While you're at it, please move this expression to a separate file in `python-modules/`